packages/mermaid/src/diagrams/pie/pieRenderer.ts

```diff
@@ -118,7 +124,15 @@ export const draw: DrawDefinition = (text, id, _version, diagObj) => {
     .enter()
     .append('text')
     .text((datum: d3.PieArcDatum<D3Sections>): string => {
-      return ((datum.data.value / sum) * 100).toFixed(0) + '%';
+      let num = datum.data.value / sum;
+
+      // "Half even" rounding mode rounds exactly .5 to the nearest, even number.
+      // By using 3 decimal places, we can round "close to .5" to the nearest even number too.
+      if (pieConfig.percentageDecimals === 0) {
+        num = Number(num.toFixed(3));
+      }
+
```

Comment:
I'm not really sure this block is a good idea, since diagrams like below will change.

E.g., if I have 4.51%, I'd expect it to round to 5%, not to 4%.

````markdown
```mermaid
pie title Pets adopted by volunteers
    "Dogs" : 5.51 %% should round up to 6
    "Cats" : 5.49 %% should round down to 5
    "Rats" : 4.51 %% should round up to 5
    "Mice" : 4.49 %% should round down to 4
    "Other": 80
```
````

```mermaid
pie title Pets adopted by volunteers
    "Dogs" : 5.51 %% should round up to 6
    "Cats" : 5.49 %% should round down to 5
    "Rats" : 4.51 %% should round up to 5
    "Mice" : 4.49 %% should round down to 4
    "Other": 80
```

Reply:
I would agree with this

Reply:
@thedustin how can we fix this?